src/custom/components/ErrorBoundary/ErrorBoundaryMod.tsx

```diff
@@ -70,6 +87,9 @@ export default class ErrorBoundary extends React.Component<unknown, ErrorBoundar
       const encodedBody = encodeURIComponent(issueBody(error))
       return (
         <FallbackWrapper>
+          <HeaderWrapper>
```

Comment:
@anxolin As you anticipated, I have some problems with the header links. Maybe because the header is inside the ErrorBoundary.
Can we try to fix that or maybe we can show only the CowSwap logo to go back to the home.

Reply:
I would simplify as u suggest. An alternative header, with just a Logo suffices.